src/components/VDialog/VDialog.js

```diff
@@ -193,14 +193,18 @@ export default {
       }, [this.$slots.activator]))
     }
 
-    const dialog = h('transition', {
-      props: {
-        name: this.transition || '', // If false, show nothing
-        origin: this.origin
-      }
-    }, [h('div', data,
-      this.showLazyContent(this.$slots.default)
-    )])
+    let dialog
+    let baseDialog = h('div', data, this.showLazyContent(this.$slots.default))
```

Comment:
What about:

```
let dialog = h('div', data, this.showLazyContent(this.$slots.default))

if (this.transition) {
  dialog = h('transition', {
    props: {
      name: this.transition,
      origin: this.origin
    }
  }, [dialog])
}
```

Reply:
Done, and tested. Thanks for the suggestion.